pkg/kubelet/container/helpers.go

```diff
@@ -133,6 +134,13 @@ func ExpandContainerCommandOnlyStatic(containerCommand []string, envs []v1.EnvVa
 	return command
 }
 
+func ExpandContainerVolumeMounts(mount *v1.VolumeMount, envs []EnvVar) {
+	mapping := expansion.MappingFuncFor(EnvVarsToMap(envs))
+	mount.SubPath = expansion.Expand(mount.SubPath, mapping)
```

Comment:
To follow the style of `ExpandContainerCommandAndArgs`, instead of modifying the mount pointer, can this return the expanded string?

Reply:
Done